lib/SILOptimizer/Mandatory/MoveKillsCopyableAddressesChecker.cpp

```diff
@@ -1895,9 +1918,10 @@ void DataflowState::init() {
 // Returns true if we emitted a diagnostic and handled the single block
 // case. Returns false if we visited all of the uses and seeded the UseState
 // struct with the information needed to perform our interprocedural dataflow.
-static bool performSingleBasicBlockAnalysis(DataflowState &dataflowState,
-                                            SILValue address,
-                                            MarkUnresolvedMoveAddrInst *mvi) {
+static bool performSingleBasicBlockAnalysis(
```

Comment:
In the later commit, I change this to be a method to remove the globals I am passing in here.